src/config.ts

```diff
@@ -22,5 +22,8 @@ export default function buildConfig(
     aqi: config.aqi ?? {},
     stats: config.stats ?? [],
     shortcuts: config.shortcuts ?? [],
+
+    // Pass other config properties
+    ...config,
```

Comment:
It's better to spread config before properties, so they won't be overridden by properties from config.